src/lib/tour-overlay/TourOverlay.component.tsx

```diff
@@ -32,8 +32,9 @@ export interface TourOverlayRef {
 
 interface TourOverlayProps {
   color: ColorValue;
-  current: number | undefined;
+  current: Optional<number>;
```

Comment:
`Optional` sounds more like the `current` prop is optional like `current?`. But not sure to have any good name to propose haha,  `Undefinable` maybe 🤔 ?

Reply:
Not a big deal tho

Reply:
Yep, but it's different to define `current?: number` and `current: number | undefined`. The second one means you must set the property even if it's undefined, which sometimes is really helpful. That's the intention of the `Optional<T>` type, I agree the name can be better, but given `Nullable<T>` is used for `null` and `Undefinable<T>` can be hard to read, I think so far `Optional<T>` is the best we have 😅